activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -178,6 +178,17 @@ def return_value_after_insert?(column) # :nodoc:
         supports_insert_returning? ? column.auto_populated? : column.auto_increment?
       end
 
+      # See https://dev.mysql.com/doc/refman/8.0/en/invisible-indexes.html for more details on MySQL feature.
+      # See https://mariadb.com/kb/en/ignored-indexes/ for more details on the MariaDB feature.
+      def supports_disabling_indexes?
+        if mariadb?
+          database_version >= "10.6.0"
+        else
+          database_version >= "8.0.0"
+        end
+      end
+      alias_method :supports_enabling_indexes?, :supports_disabling_indexes?
```

Comment:
I don't see the value in exposing two methods that do the same thing. Can we get rid of the alias?